src/components/SignInForm.jsx

```diff
@@ -2,199 +2,216 @@ import React, { useState } from "react";
 import { Formik, Form, Field } from "formik";
```

Comment:
@tmccoy529 we may need to do these pr's in two steps

- 1 being submit a pr for the formatting of the component or file you wish to ----------
- 2 being the acutal changes

Reply:
So Im not sure what you are talking about here. I made changes to buttons and modals. All this other stuff is clearly formatting changes. I can back out all these format changes if that makes it easier to read. I dont even know why they updated other than we were both playing with the formatting since it had been an issue earlier and I guess it got carried over?

Reply:
So this is just a recommendation going forward, it's clear that formatting will be updated for almost all of these changes.

So what I'm suggesting when you decided what you are going to work. Pick a couple files or whatever that need to change. Save all of them with the new formatting and immediately submit that for pr. 

Once approved, you can make your changes and it will be easier to review because you will only see the changed code.

When it's both the formatting and changes, its very hard to tell what changed.

Does that make sense?

Reply:
The orginal quesiton, was just what lines were the actual non formatted changes. Like converting a button to a dotgov components Button

Reply:
Yeah I get it. Why dont I just do a full format PR. Get it all over with then this wont happen again.

Reply:
@tmccoy529 that works for me, was trying to save you having to do that, but i guess it probably makes sense.

Reply:
@martypowell yeah it was a pain but I think its worth it. Any file we change at this point is going to have the same problem where changes are lost in the giant format update so its better to just knock it all out now.

Reply:
sweet, the other pr is approved.